### PR TITLE
Update show.html.erb

### PR DESCRIPTION
**Pull request by avrildavid**
Updating the Certification FAQs based on our call with Elisabeth Maher in the Appeals Management Office. Adding relevant images to the asset library.

### CODE REVIEW

File: app/views/help/show.html.erb [hunk 1]
```diff
@@ -35,8 +35,7 @@
   <h2 id="what-is-caseflow">1. What is Caseflow Certification?</h2>
 
   <p>
-    Caseflow Certification, a web-based tool that makes it easy to automatically
-    fill out an electronic Form 8 for paperless appeals. The tool was built by the
+    Caseflow Certification, is a web-based tool that pre-fills the electronic Form 8 for paperless appeals and checks case documents for readiness for certification. The tool was built by the
```
Comment (amprokop) [inline app/views/help/show.html.erb]: Looks like a typo, the comma here should be removed i.e. "Caseflow Certification is a web-based tool"

File: app/views/help/show.html.erb [hunk 2]
```diff
@@ -132,6 +131,14 @@
     listed below, you should add correct label to the document in the VBMS document
     settings and click the refresh button in Caseflow.
   </p>
+  
+  <h3>3. If a document is missing from the eFolder, the appeal cannot be certified using Caseflow and should instead be certified manually.</h3>
+
+  <p>
+    If a document is missing a label or uses something other than one of the labels
+    listed below, you should add correct label to the document in the VBMS document
```
Comment (amprokop) [inline app/views/help/show.html.erb]: Maybe "add the correct label"

File: app/views/help/show.html.erb [hunk 3]
```diff
@@ -200,16 +206,18 @@
     The Form 8 will automatically be uploaded to the eFolder, a certification date
     will be added to the appeal, and the appeal will be certified to the Board of
     Veterans Appeals!  Once you see the "Congratulations" page you can close the
-    browser window and open up another appeal using VACOLS.
+    browser window and open up another appeal using VACOLS. Please keep in  mind that you should
```
Comment (amprokop) [inline app/views/help/show.html.erb]: Extra space here: "Please keep in mind"

File: app/views/help/show.html.erb [hunk 4]
```diff
@@ -200,16 +206,18 @@
...
+    continue to follow existing procedures for dispatching the Form 8 to the Board and sending
+    the notification letter to the claimant, the "File Certified to BVA" letter, after the Form 8
```
Comment (amprokop) [inline app/views/help/show.html.erb]: maybe  `(the "File Certified to BVA" letter)` instead of `, the "File Certified to BVA" letter,`?

**Comment by amprokop** [comment]
Sorry if i jumped in on this too early! I was waiting for something else to finish and just noticed your PR. The FAQ changes look great and I think they'll help folks understand how to use Certification much better :)

File: app/views/help/show.html.erb (hunk 1 above)
Comment (avrildavid) [inline app/views/help/show.html.erb]: Agreed - happy to make this change

File: app/views/help/show.html.erb (hunk 2 above)
Comment (avrildavid) [inline app/views/help/show.html.erb]: Works for me

File: app/views/help/show.html.erb (hunk 3 above)
Comment (avrildavid) [inline app/views/help/show.html.erb]: Will edit

File: app/views/help/show.html.erb (hunk 4 above)
Comment (avrildavid) [inline app/views/help/show.html.erb]: Also works for me. Thank you for getting to this so quickly and working with me on it! :)

**Comment by avrildavid** [comment]
@amprokop it looks like the images aren't displaying for FAQs 7 and 11. It just has the descriptive text " 7- add correct doc label in vbms" and "11 - clear certification date in vacols". I saved the corresponding images in the asset folder - let me know if there's anything I can do to correct this. Thank you!

**Pull request merged**